src/core/timer.c

```diff
@@ -391,8 +392,9 @@ static void timer_enter_waiting(Timer *t, bool time_change) {
                          * if the scheduled time has already passed, set the time when systemd
                          * first started as the scheduled time.
                          * Also, we don't have to check t->persistent since the logic implicitly express true. */
```

Comment:
Maybe remove this comment about `t->persistent` since this also applies to non-persistent timers?

Reply:
Done!